aten/src/ATen/native/mps/TensorFactory.cpp

```diff
@@ -101,6 +101,25 @@ Tensor empty_strided_mps(
   return t;
 }
 
+Tensor empty_strided_symint_mps(
+    SymIntArrayRef size,
+    SymIntArrayRef stride,
+    c10::optional<ScalarType> dtype_opt,
+    c10::optional<Layout> layout_opt,
+    c10::optional<Device> device_opt,
+    c10::optional<bool> pin_memory_opt) {
+  check_size_nonnegative(size);
+  // empty memory formatempty
+  auto t = at::native::empty_mps(
+      {0},
+      dtype_opt,
+      layout_opt,
+      device_opt,
+      pin_memory_opt);
+  resize_impl_mps_(t.unsafeGetTensorImpl(), c10::asIntArrayRefSlow(size), c10::asIntArrayRefSlow(stride));
+  return t;
```

Comment:
I expected this to call `empty_strided_mps` directly

Reply:
Yep that's fair. This mostly just replicates the logic of `empty_strided_mps` (uglier)